src/components/Table/index.js

```diff
@@ -74,9 +131,10 @@ Table.propTypes = {
     data: PropTypes.arrayOf(Object).isRequired,
     /**
      * Specifies the sorting direction, valid options are 'asc' or 'desc'.
-     * @ignore
      */
     sortDirection: PropTypes.oneOf(['asc', 'desc']),
+    /** Action triggered when a column header is clicked */
```

Comment:
Action triggered when a column is sorted. Receive the event object, field and sortDirection.